sos/policies/runtimes/__init__.py

```diff
@@ -55,7 +55,7 @@ def load_container_info(self):
         self.images = self.get_images()
         self.volumes = self.get_volumes()
 
-    def check_is_active(self):
+    def check_is_active(self, sysroot=None):
```

Comment:
From my tests it looks like only passing sysroot here isn't enough, since get_containers, get_images, ... functions are not using this sysroot and therefore can't get right output

Reply:
Ack, I think I see where this breaks.